packages/nimble-components/src/table-column/mixins/placeholder.ts

```diff
@@ -0,0 +1,32 @@
+import { attr } from '@microsoft/fast-element';
+import type { TableColumn } from '../base';
+
+// Pick just the relevant properties the mixin depends on (typescript complains if the mixin declares private / protected base exports)
+// Because the 'placeholder' mixin doesn't depend on any properties of the TableColumn, there are no properties to pick.
+type TableColumnWithPlaceholder = Pick<TableColumn, never>;
+// prettier-ignore
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Looking at this comment again, I think you mean on line 9. Is that right? If so, no, that doesn't work because then a TypeScript error occurs: _A mixin class must have a constructor with a single rest parameter of type 'any[]'._

This isn't the prettiest code, but it's all following our established patterns for mixins, which are already used in `fractional-width-column.ts` and `groupable-column.ts`.

Reply:
Yeah, I was pointing at the suppression, thinking we might be able to use `unknown` instead of suppressing.